.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  release:
+    branches:
+      - main
+    types: [created]
+name: Publish release
+jobs:
+  release_linux:
```

Comment:
I would suggest a build matrix instead: https://github.com/ctron/hat/blob/master/.travis.yml#L9

Reply:
I think we can add the matrix tag when we work on cross-compilation.